python/tank/bootstrap/manager.py

```diff
@@ -120,14 +120,37 @@ def __repr__(self):
 
         repr  = "<TkManager "
         repr += " User %s\n" % self._sg_user
-        repr += " Cache fallback path %s\n" % self._bundle_cache_fallback_paths
+        repr += " Cache fallback path %s\n" % self._get_bundle_cache_fallback_paths()
         repr += " Caching policy %s\n" % self._caching_policy
         repr += " Plugin id %s\n" % self._plugin_id
         repr += " Config %s %s\n" % (identifier_type, self._pipeline_configuration_identifier)
         repr += " Allows config overrides %s\n" % self._allow_config_overrides
         repr += " Base %s >" % self._base_config_descriptor
         return repr
 
+    def _get_bundle_cache_fallback_paths(self):
+        """
+        Retuns a list containing both the user specified bundle caches and the ones specified by
+        Toolkit.
+
+        :returns: List of bundle cache paths.
+        """
+        if constants.BUNDLE_CACHE_FALLBACK_PATHS_ENV_VAR in os.environ:
+            fallback_str = os.environ[constants.BUNDLE_CACHE_FALLBACK_PATHS_ENV_VAR]
+            log.debug(
+                "Detected %s environment variable set to '%s'" % (
+                    constants.BUNDLE_CACHE_FALLBACK_PATHS_ENV_VAR,
+                    fallback_str
+                )
+            )
+            toolkit_bundle_cache_fallback_paths = fallback_str.split(os.pathsep)
+        else:
+            toolkit_bundle_cache_fallback_paths = []
+
+        return list(
+            set(self._user_bundle_cache_fallback_paths + toolkit_bundle_cache_fallback_paths)
```

Comment:
The global bundle cache is not part of that bundle cache list. However, its clear from my use of set that it doesn't preserve the order of insertion. I'll update the code accordingly.